controllers/storageclassclaim/storageclassclaim_controller.go

```diff
@@ -638,9 +656,18 @@ func (r *StorageClassClaimReconciler) reconcileCephFilesystemSubVolumeGroup() er
 		klog.Warningf("More than one CephFileSystem found in the cluster, selecting the first one")
 	}
 
-	cephFileSystemName = cephFilesystemList.Items[0].Name
+	parameters, err := storagecluster.GenerateCephFSProviderParameters(r.storageCluster)
+	if err != nil {
+		return err
+	}
+
+	cephFileSystem := cephFilesystemList.Items[0]
+	cephFileSystemName = cephFileSystem.Name
+	for _, cfs := range cephFileSystem.Spec.DataPools {
+		cfs.PoolSpec.Parameters = parameters
+	}
```

Comment:
Why are we applying it here and not in the code that creates the `cephFilesystem` in the storagecluster controller? Anyhow, setting anything on `cephFilesystem` here will not be applied as this function is only reconciling the `CephFilesystemSubVolumeGroup` resource.